index.js

```diff
@@ -1,216 +1,413 @@
 /*
+
```

Comment:
Were all of the newlines intentional?
(Note: if it is, there are some double newlines. Not sure if it was intentional)

Reply:
Something went wrong with my VSCode Style Config, let me fix that